src/sentry/integrations/github/issues.py

```diff
@@ -26,6 +27,16 @@ def get_issue_url(self, key: str) -> str:
         repo, issue_id = key.split("#")
         return f"https://{domain_name}/{repo}/issues/{issue_id}"
 
+    def get_feedback_issue_body(self, event: GroupEvent) -> str:
+        body = "|  |  |\n"
+        body += "| ------------- | --------------- |\n"
+        for evidence in sorted(
+            event.occurrence.evidence_display, key=attrgetter("important"), reverse=True
+        ):
+            body += f"| **{evidence.name}** | {evidence.value} |\n"
+
+        return body[:-2]
```

Comment:
maybe a comment why the last two least important are dropped?

Reply:
this is copied from the `get_generic_issue_body` function that already existed, so i don't have the full context, but i'm thinking it's to remove the newline and (maybe unintentionally?) the last pipe `|` in the body. i'm going to change this to just truncate the body by 1, so that we only chop off the newline